apps/api/src/services/scrapers/zhihu/content_processing.py

```
@@ -0,0 +1,64 @@
+    soup = BeautifulSoup(html, "html.parser")
+    for a_tag in soup.find_all("a", href=True):
+        href = a_tag["href"]
+        if href.startswith("https://link.zhihu.com/"):
+            try:
+                parsed = urlparse(href)
+                qs = parse_qs(parsed.query)
+                target = qs.get("target", [None])[0]
+                if target:
+                    a_tag["href"] = unquote(target)
+            except Exception:
+                pass
```

Ruff (0.15.6)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)